include/fs/fs.h

```diff
@@ -170,20 +182,32 @@ struct fs_file_system_t {
 #define FS_SEEK_END	2	/* Seek from end of file.  */
 #endif
 
-
 /**
- * @brief File open
+ * @brief File open extended
```

Comment:
I'm not sure adding word "extended" is useful here. (If this is extended, that implies somewhere there's "barebones" one, but there's no such. Indeed, having flags fro open is a "baseline". The fact that we didn't have them before is, well, a bug.)

Reply:
Will fix. This is leftover from having two fs_open functions, the documentation for the one supporting the flags have been left, but description has not been fixed.

Reply:
Done.